src/tool/subcommands/snapshot_cmd.rs

```diff
@@ -44,6 +44,23 @@ pub enum SnapshotCommands {
         vendor: snapshot::TrustedVendor,
     },
 
+    /// Validates the snapshot.
```

Comment:
Let's document how this subcommand is different from the other one.

Reply:
I have changed the document of `validate` command.